backend/src/main/java/org/eclipse/tractusx/puris/backend/common/edc/logic/service/EdcAdapterService.java

```diff
@@ -104,14 +124,19 @@ private Response sendPostRequest(JsonNode requestBody, List<String> pathSegments
         for (var pathSegment : pathSegments) {
             urlBuilder.addPathSegment(pathSegment);
         }
-        RequestBody body = RequestBody.create(requestBody.toString(), MediaType.parse("application/json"));
+        RequestBody body = null;
+
+        if (requestBody != null) {
+            body = RequestBody.create(requestBody.toString(), MediaType.parse("application/json"));
+        }
```

Comment:
Fair, had a look earlier on that at StackOverflow because there were EDC requests that need empty body, but I see that I didn't check it for the latest versions. You're right, there isn't the null post anymore. Will revert that.